### ISSUE TITLE
delay on ActivateDoor

### ISSUE THREAD

**Issue by aSmallChild** [jcox10/ESP8266-GarageDoorMonitor#1]
I noticed you used a ticker for CheckDoorStatus  and a delay() in ActivateDoor.

CheckDoorStatus can be run instantly so it doesn't need to be on a ticker instead of in the loop. Whereas having a delay of one second in your ActivateDoor function means that your http responses are slow. If you're interested in trying it out:

void ActivateDoor() {
  Serial.println("Turning relay on");
  digitalWrite(relayPin, HIGH);
  _ticker.attach_ms(500, ReleaseDoorButton);
}

void ReleaseDoorButton() {
  Serial.println("Turning relay off");
  digitalWrite(relayPin, LOW);
  _ticker.detach();
}

**Comment by jcox10** [comment]
Thanks for this! Makes much more sense than having the delay.
